crates/swc/tests/errors/lints/no-use-before-define/check-let-and-const/.swcrc

```diff
@@ -0,0 +1,7 @@
+{
+    "jsc": {
+        "lints": {
+            "noUseBeforeDefine": ["off"]
```

Comment:
Why does this lint work when it's `off`?

Reply:
let and const checks regardless of enabled or disabled rule. I did it on purpose. Of course I can keep empty lints config, but I decide to disable rule visibly